themes/atomic/atomic.theme.bash

```diff
@@ -31,175 +28,175 @@ Face="\342\230\273"
+  color=${IRed}
+  box="[|]${normal}${Line}"
+  info="\w"
+  printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
 }
 
 ___atomic_prompt_scm() {
-	[ "${THEME_SHOW_SCM}" != "true" ] && return
-	color=$bold_green
-	box="[${IWhite}$(scm_char)] "
-	info="$(scm_prompt_info)"
-	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
+  [ "${THEME_SHOW_SCM}" != "true" ] && return
+  color=$bold_green
+  box="[${IWhite}$(scm_char)] "
+  info="$(scm_prompt_info)"
+  printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
 }
 
 ___atomic_prompt_python() {
-	[ "${THEME_SHOW_PYTHON}" != "true" ] && return
-	color=$bold_yellow
-	box="[|]"
-	info="$(python_version_prompt)"
-	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_blue}" "${box}"
+  [ "${THEME_SHOW_PYTHON}" != "true" ] && return
+  color=$bold_yellow
+  box="[|]"
+  info="$(python_version_prompt)"
+  printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_blue}" "${box}"
 }
 
 ___atomic_prompt_ruby() {
-	[ "${THEME_SHOW_RUBY}" != "true" ] && return
-	color=$bold_white
-	box="[|]"
-	info="rb-$(ruby_version_prompt)"
-	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_red}" "${box}"
+  [ "${THEME_SHOW_RUBY}" != "true" ] && return
+  color=$bold_white
+  box="[|]"
+  info="rb-$(ruby_version_prompt)"
+  printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_red}" "${box}"
 }
 
 ___atomic_prompt_todo() {
-	[ "${THEME_SHOW_TODO}" != "true" ] ||
-	[ -z "$(which todo.sh)" ] && return
-	color=$bold_white
-	box="[|]"
-	info="t:$(todo.sh ls | egrep "TODO: [0-9]+ of ([0-9]+)" | awk '{ print $4 }' )"
-	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_green}" "${box}"
+  [ "${THEME_SHOW_TODO}" != "true" ] ||
+  [ -z "$(which todo.sh)" ] && return
+  color=$bold_white
+  box="[|]"
+  info="t:$(todo.sh ls | egrep "TODO: [0-9]+ of ([0-9]+)" | awk '{ print $4 }' )"
+  printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_green}" "${box}"
 }
 
 ___atomic_prompt_clock() {
-	[ "${THEME_SHOW_CLOCK}" != "true" ] && return
-	color=$THEME_CLOCK_COLOR
-	box="[|]"
-	info="$(date +"${THEME_CLOCK_FORMAT}")"
-	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
+  [ "${THEME_SHOW_CLOCK}" != "true" ] && return
+  color=$THEME_CLOCK_COLOR
+  box="[|]"
+  info="$(date +"${THEME_CLOCK_FORMAT}")"
+  printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
 }
 
 ___atomic_prompt_battery() {
-	[ ! -e "$BASH_IT"/plugins/enabled/battery.plugin.bash ] ||
-	[ "${THEME_SHOW_BATTERY}" != "true" ] && return
-	batp=$(battery_percentage)
-	if [ "$batp" -gt 50 ]; then
-		color=$bold_green
-	elif [ "$batp" -lt 50 ] && [ "$batp" -gt 25 ]; then
-		color=$bold_yellow
-	elif [ "$batp" -lt 25 ]; then
-		color=$IRed
-	fi
-	box="[|]"
-	ac_adapter_disconnected && info="-"
-	ac_adapter_connected && info="+"
-	info+=$batp
-	[ "$info" == "+100" ] && info="AC"
-	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
+  [ ! -e "$BASH_IT"/plugins/enabled/battery.plugin.bash ] ||
```

Comment:
@nwinkler 
> You shouldn't rely on the name and location of the enabled battery plugin. See my inline comment.
- Yes, So I'll check the function of the plugin instead of this in a few moments i will go up the changes in the code :stuck_out_tongue_winking_eye:

> In general, it's not great to mix functional changes and whitespace changes. Due to all of the reformatting you did in the PR, it's hard to see the real changes. It's typically better to do these in two separate changes.

I'm sorry i used atom and beautify plugin and I did not realize what happened, this will not happen again, I will take your recommendation for futures pulls